maven/index.js

```diff
@@ -11,6 +11,7 @@ const password = core.getInput('password');
 const name = core.getInput('name');
 const group = core.getInput('group');
 const buildDir = core.getInput('buildDir');
+const includeDotFiles = core.getInput('includeDotFiles');
```

Comment:
maybe getter for boolean?
```suggestion
const includeDotFiles = core.getBooleanInput('includeDotFiles');
```

Reply:
Good catch, fixed!